demo/app/components/validation/validation.component.html

```diff
@@ -197,28 +197,28 @@ <h3 tsCardTitle tsVerticalSpacing="small--0">Contains numbers</h3>
     <h3 tsCardTitle>Url</h3>
 
     <div fxLayout="row" [fxLayoutGap]="flexGap">
-<pre fxFlex>
-// Valid URLs:
-http://foo.com
-http://foo.com/blah_blah_(wikipedia)
-https://www.example.com/foo/?bar=baz&inga=42&quux
-http://userid:password@example.com:8080/
-http://⌘.ws
-ftp://foo.bar/baz
-http://foo.bar/?q=Test%20URL-encoded%20stuff
-http://223.255.255.254
-</pre>
-
-<pre fxFlex>
-// Invalid URLs:
-http://
-http://foo.bar?q=Spaces should be encoded
-foo.com
-rdar://1234
-http://1.1.1.1.1
-http://.www.foo.bar./
-http:// shouldfail.com
-</pre>
+      <pre fxFlex>
```

Comment:
We should undo this change.. I know it makes indentation super odd, but the `pre` element will always respect indentation. So this will actually output 3 indentions in the UI and we really want it to be at 0 indentation when output.